ui/src/views/network/CreateVpc.vue

```diff
@@ -359,6 +363,28 @@ export default {
         }
       })
     },
+    fetchOwnerOptions (OwnerOptions) {
+      this.owner = {
+        projectid: null,
+        domainid: this.$store.getters.userInfo.domainid,
+        account: this.$store.getters.userInfo.account
+      }
+      if (OwnerOptions.selectedAccountType === 'Account') {
+        if (!OwnerOptions.selectedAccount) {
+          return
+        }
+        this.owner.account = OwnerOptions.selectedAccount
+        this.owner.domainid = OwnerOptions.selectedDomain
+        this.owner.projectid = null
+      } else if (OwnerOptions.selectedAccountType === 'Project') {
```

Comment:
There is a soft dependency in lines 372 and 379 to PR #10052.

Reply:
soft meaning; it will look ugly but still work, right?

Reply:
In the meaning that it will work for the English locale, any other language, this PR will fail without the changes in #10052.

Reply:
yeah, sorry for my bad English. That I meant.